playpen/ansible/roles/dev/files/bashrc

```diff
@@ -72,6 +72,8 @@ _paction() {
 
 psmash() {
     workon pulp-smash;
+    prestart;
+    find ~/devel/pulp-smash/ -name *.pyc -delete;
```

Comment:
It might be good to swap the order of these two lines, and have it rm py[co]'s from ~/devel instead of just smash. Or perhaps even better, why not modify prestart itself to rm the py[co]'s from ~/devel/pulp*? That might be nice. It could be a separate PR if you prefer, or if you don't want to do it let me know and I'll make the PR.